recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend

```diff
@@ -5,7 +5,7 @@ SRCREV = "${AUTOREV}"
 LINUX_VERSION = "4.14.75"
 
 SRC_URI = " \
-    git://github.com/xen-troops/linux.git;branch=${BRANCH} \
+    git://github.com/otyshchenko1/linux.git;branch=${BRANCH} \
```

Comment:
Is it ok to use personal repo?

Reply:
That patch is not for merge as it's subject actually claims, but for possibility to build with required changes to Linux applied and minimal integration effort. This patch will be droped when corresponding PR to Linux goes in.

Reply:
Already merged https://github.com/xen-troops/linux/pull/81 and updated current PR by removing a commit in question.